package.json

```diff
@@ -51,7 +51,7 @@
     "ts": "node -r esbuild-register",
     "typecov": "type-coverage",
     "update": "yarn ts ./tools/update.ts && yarn format-for-gen-file",
-    "version": "env-cmd -e version yarn update && git add ."
+    "version": "env-cmd -e version yarn update && changeset version"
```

Comment:
Like `prerelease`,  we can extract `preversion`,  but it's not required, of course.

Reply:
We can't use preversion because the script needs to be run after package.json is updated.

Reply:
Then or `postversion`?